5.sc_Morphology_Logistic_Regression/0.split_data/0.split_data_binary_classification.ipynb

```diff
@@ -0,0 +1,851 @@
+{
```

Comment:
<p>I could see this being multiple analysis, I kind of like performing each analysis in isolation, but up to you. This will also depend on how often code is reused, or if you plan to automate many different analysis slightly</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/pyroptosis_signature_data_analysis/pull/58/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>